lib/resolvers.js

```diff
@@ -38,6 +70,56 @@ const Resolvers = {
     }
   },
   Mutation: {
+    createAccount: async (root, { id, isAdmin }, request) => {
+      await validateAdminAccess(request);
+      const sql = 'CALL create_account(?, ?)';
+      const pool = request.server.app.mysql;
+      const results = await Common.doQuery(sql, [id, isAdmin ? 1 : 0], pool);
```

Comment:
If the `isAdmin ? 1 : 0` statements are hurting code coverage, you could do `+isAdmin`.